src/experiment-apis/matchPatterns.json

```diff
@@ -10,7 +10,7 @@
         "parameters": [
           {
             "name": "patterns",
-            "description": "Array of string URL patterns to whitelist",
+            "description": "Array of string URL patterns to safelist",
```

Comment:
As I said in https://github.com/mozilla-mobile/android-components/pull/8642#discussion_r505816513, there is no security-context here, so "safelist" is actually inaccurate. "allowlist" or simply rewording "Array of string URL patterns to match" is much more accurate here.